map/src/main/res/layout/marker_gallery_number.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/ideal_sans_medium"
+        android:textColor="@color/white"
+        android:textSize="15sp" />
```

Comment:
harcoded text size. can we use a textstyle?

Reply:
we can but it's only 1 place used so no real reason (no need to add a style if it's only used once is there?)

Reply:
eh yeah I think so because even so its easier to track. It should be part of zeplin even if its used once. So if it changes in the future we just need to change the style, not find exactly where the view is.

Reply:
yea ight I'll make a style, and hopefully zeplin will update it.